cmd/aterm/config/config.go

```diff
@@ -1,154 +1,229 @@
-// Copyright 2020, Verizon Media
-// Licensed under the terms of the MIT. See LICENSE file in project root for terms.
-
 package config
 
 import (
-	"encoding/base64"
-	"flag"
+	"fmt"
 	"io"
 	"io/ioutil"
+	"net/url"
 	"os"
 	"path/filepath"
 
+	"github.com/OpenPeeDeeP/xdg"
 	"github.com/kelseyhightower/envconfig"
-	"github.com/pkg/errors"
+	"github.com/theparanoids/aterm/errors"
+	"github.com/theparanoids/aterm/fancy"
+	"github.com/theparanoids/aterm/network"
 	"gopkg.in/yaml.v2"
+
+	"github.com/hashicorp/go-multierror"
 )
 
-var configFilePath = filepath.Join(os.Getenv("HOME"), ".config", "ashirt", "term-recorder.yaml")
+func configHome() string {
+	return filepath.Join(xdg.ConfigHome(), "ashirt")
```

Comment:
Revised, with a small caveat. For ASHIRT, we ended up using the data directory everywhere, which places thing, on a mac, in the `Library/Application Support` directory structure. On Linux, it places it in `~/.local/share` (Rather than `~/.config`). I opted to do the same thing for this application, as I wanted both applications to reflect the same structure.